src/pages/commits/ActiveCommits/BuildVariantCard.tsx

```diff
@@ -67,34 +89,52 @@ const RenderGroupedIcons = ({ tasks }) => {
   );
   return (
     <>
-      {otherTasks.map(({ count, umbrellaStatus }) => (
-        <GroupedTaskStatusBadgeWrapper>
-          <GroupedTaskStatusBadge
-            status={umbrellaStatus}
-            key={`${umbrellaStatus}_groupedBadge`}
-            count={count}
-          />
-        </GroupedTaskStatusBadgeWrapper>
-      ))}
+      {otherTasks.map(({ count, umbrellaStatus }) => {
+        const onClick = () =>
+          push(
+            getVersionRoute(versionId, {
+              statuses: mapUmbrellaStatusToQueryParam[umbrellaStatus],
+              variant: buildVariantDisplayName,
```

Comment:
This generated a broken link http://localhost:3000/version/mongodb_mongo_master_4a43e817011e7cde84c2dce87619d81921f19c9a/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC&statuses=running-umbrella,started,dispatched&variant=%2A%20Windows%20DEBUG 
Which did not return a usable page since the variant did not match the input. 
![image](https://user-images.githubusercontent.com/4605522/134273100-684568c8-c623-48dc-8a1b-dd55c6d8c253.png)

We should use a strict regex and match against the variant name instead of the display name since the variant name is guaranteed to be unique. 
Something similar to what we do here https://github.com/evergreen-ci/spruce/blob/ae1f4d36bb82735479366507f625761386dd46e2/src/pages/version/buildVariants/GroupedTaskSquare.tsx#L28-L32